tcp-to-blob/package.json

```diff
@@ -10,6 +10,7 @@
     "deploy": "yarn docker-push && yarn deploy-fast",
     "deploy-dashboard": "yarn make-env-files && az portal dashboard create --resource-group \"${AZ_RESOURCE_GROUP}\" --name \"${NAME_PREFIX}-dashboard\" --input-path \"./dist/env/${NAME_PREFIX}/dashboard.json\" --location \"${AZ_LOCATION}\"",
     "run-canary": ". ./deploy/env-defaults.sh && echo \"Sending test data to ${AKS_NAME}.\" && yarn make-env-files && kubectl delete --ignore-not-found=true -f ./dist/env/${NAME_PREFIX}/tcp-to-blob-canary.yaml && kubectl apply -f ./dist/env/${NAME_PREFIX}/tcp-to-blob-canary.yaml",
+    "run-canary-cron": ". ./deploy/env-defaults.sh && echo \"Sending test data to ${AKS_NAME}.\" && yarn make-env-files && kubectl delete --ignore-not-found=true -f ./dist/env/${NAME_PREFIX}/tcp-to-blob-canary-cron.yaml && kubectl apply -f ./dist/env/${NAME_PREFIX}/tcp-to-blob-canary-cron.yaml",
```

Comment:
You may wish to update the `echo` message to reflect that it's deploying a CRON job to send test data.